src/allotropy/parsers/agilent_gen5/agilent_gen5_structure.py

```diff
@@ -238,9 +271,8 @@ def create(cls, lines: list[str]) -> list[ReadData]:
         read_type = cls.get_read_type(procedure_details)
         if read_type != ReadType.ENDPOINT:
             raise AllotropeConversionError(UNSUPPORTED_READ_TYPE_ERROR)
-
         read_modes = cls.get_read_modes(procedure_details)
-        read_sections = list(SectionLinesReader(lines).iter_sections("^Read\t"))
+        read_sections = list(SectionLinesReader(lines).iter_sections(r"^\s{0,4}Read\t"))
```

Comment:
I'm guessing `^\s{0,4}` is to match a variable number of whitespace. Let's do `^\s*` to cover all cases.